pkgpanda/util.py

```diff
@@ -226,11 +301,17 @@ def expect_fs(folder, contents):
 
 
 def make_tar(result_filename, change_folder):
-    tar_cmd = ["tar", "--numeric-owner", "--owner=0", "--group=0"]
+    if is_windows:
+        tar_cmd = ["bsdtar"]
+    else:
+        tar_cmd = ["tar", "--numeric-owner", "--owner=0", "--group=0"]
     if which("pxz"):
         tar_cmd += ["--use-compress-program=pxz", "-cf"]
     else:
-        tar_cmd += ["-cJf"]
+        if is_windows:
+            tar_cmd += ["-cf"]
+        else:
+            tar_cmd += ["-cJf"]
```

Comment:
I will see what I can do

Reply:
one option is to go to a libArchive library so we are not calling out to the command line.

Reply:
I'd be open to that if you'd like to explore it. That change should be well covered by integration tests (if we're producing bad tarballs, lots of stuff should fail), so it might be worth trying.

Reply:
this is something I do not have time to do but if you do it will give better parity in the future if done, and will stop having to call out to an external command.

Reply:
also, there is now only 1 if is_windows in the function as I removed the other.

Reply:
still investigating this one. apparently I am using an old version of bsdtar!